tasks/syntaxPractice.go

```diff
@@ -1,31 +1,52 @@
 package tasks
 
+import "fmt"
+
 // Task 1: Write a function that prints "Hello World" to the console.
 
 func PrintHelloWorld() {
+	fmt.Println("Hello World!")
+
 	// Implement the code to print "Hello World"
 }
 
 // Task 2: Write a function that takes an integer and prints it to the console using string formatting.
-
 func PrintInteger(n int) {
+	var num int
+	fmt.Scanln(&num)
+	fmt.Println(num)
 	// Implement the code to print the integer n.
 }
 
 // Task 3: Write a function that takes two integers, adds them, and prints the result.
 
 func PrintSum(a, b int) {
 	// Implement the code to print the sum of a and b.
+	var num1, num2 int
+	fmt.Println(num1 + num2)
```

Comment:
No need to take input for num1 and num 2, just implement the code to print sum of int a and b passed as an argument to the function.